src/STAT_FrontEnd.C

```diff
@@ -103,28 +108,11 @@ STAT_FrontEnd::STAT_FrontEnd()
         setenv("MRNET_DEBUG_LOG_DIRECTORY", envValue, 1);
 
     /* Set MRNet and LMON rsh command */
-    envValue = getenv("STAT_LMON_REMOTE_LOGIN");
```

Comment:
Moved to STAT_lmonFrontEnd::STAT_lmonFrontEnd  with the rest of the _LMON_ environment variables.